recursion/get_maze_path.js

```diff
@@ -0,0 +1,22 @@
+const get_maze_path = (sr, sc, dr, dc) => {
```

Comment:
@arc-arnob  logic looks good. But even during the practice session, make sure to give meaningful names to the variable. 

Good habit to follow. :)